helper/packager.php

```diff
@@ -267,4 +268,16 @@ protected function get_template_engine()
 
 		return $template_engine;
 	}
+
+	protected function get_language_object($min_version)
+	{
+		$phpbb31 = (bool) preg_match('/^[\\D]*3\\.1.*$/', $min_version);
+
+		return array(
+			'class'		=> $phpbb31 ? '\phpbb\user' : '\phpbb\language\language',
+			'service'	=> $phpbb31 ? 'user' : 'language',
+			'function'	=> $phpbb31 ? 'add_lang_ext' : 'add_lang',
+			's_32'		=> !$phpbb31,
```

Comment:
I would prefer not to have this. I think we can just use `{% if LANGUAGE.service == 'language'` %} as a conditional test.